src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -633,10 +634,9 @@ discover_dfuse(void)
 	num_dfs = 0;
```

Comment:
mnt_type_fuse could be a #define rather than a stack variable here, and this function would be a good place to use `struct dfs_mt *` rather than `dfs_list[num_dfs].dfs_dir_hash`

Reply:
Fixed as you suggested. Thank you!